pkg/metrics/metrics.go

```diff
@@ -136,14 +136,13 @@ func (pc *PrometheusNodeConfig) Configure() error {
 		log.Info("install the prometheus-operator to enable Prometheus configuration")
 		return nil
 	}
-	// get list of Windows nodes that are in Ready phase
```

Comment:
>WMCO was made to no longer check for a Node's schedulable state when
determining if the Node's address should be added to the
windows-exporter metrics endpoint, as it has no control over the
schedulability, and due to timing issues, could potentially cause a Node
to not be added to the endpoint. This check was already problematic, as
if a user cordoned and uncordoned a node for maintenance reasons, it
could be removed from the metrics list, and not re-added until WMCO
reconciled a node.

You are actually fixing a bug here. Not sure if this should be included as part of this change. 

Would adding a node that is cordoned or not ready to the metrics endpoint cause any issues?

Reply:
I felt this was appropriate here as cordoning is no longer under WMCO's control and thus should be removed from this check. I can split this off into a separate PR.

> Would adding a node that is cordoned or not ready to the metrics endpoint cause any issues?

Cordoning definitely shouldn't. I don't see why it being ready or not should, the prometheus server is ran by windows-exporter and the metrics collector will just time out when trying to reach it. But that is a normal situation to be handled.

Reply:
>I felt this was appropriate here as cordoning is no longer under WMCO's control and thus should be removed from this check. I can split this off into a separate PR.

What does cordoning ownership have to do with this? This is a bug that would have to be fixed irrespective of cordoning ownership.